### ISSUE TITLE
Conda env create not working

### ISSUE THREAD

**Issue by Pixelatory** [KrishnaswamyLab/ReLSO-Guided-Generative-Protein-Design-using-Regularized-Transformers#1]
When I type in the command as instructed in how to run, I get this error:

Warning: you have pip-installed dependencies in your environment file, but you do not list pip itself as one of your conda dependencies.  Conda may not use the correct pip to install your packages, and they may end up in the wrong place.  Please add an explicit pip dependency.  I'm adding one for you, but still nagging you.
Collecting package metadata (repodata.json): done
Solving environment: failed

ResolvePackageNotFound:
  - libcxx==12.0.0=h2f01273_0
  - python==3.10.4=hdfd78df_0
  - openssl==1.1.1q=hca72f7f_0
  - ncurses==6.3=hca72f7f_3
  - readline==8.1.2=hca72f7f_1
  - bzip2==1.0.8=h1de35cc_0
  - ca-certificates==2022.07.19=hecd8cb5_0
  - xz==5.2.5=hca72f7f_1
  - libffi==3.3=hb1e8313_2
  - zlib==1.2.12=h4dc903c_2
  - sqlite==3.38.5=h707629a_0
  - tk==8.6.12=h5d9f67b_0

**Comment by Pixelatory** [comment]
In case anyone is interested, I got the environment working and this is my environment.yml:

name: relso_env
channels:
  - conda-forge
  - defaults
dependencies:
  - boost=1.74.0=py39hefe7e4c_5
  - boost-cpp=1.74.0=h9f4b32c_8
  - brotli=1.0.9=h8ffe710_7
  - brotli-bin=1.0.9=h8ffe710_7
  - bzip2=1.0.8=h8ffe710_4
  - ca-certificates=2022.07.19=haa95532_0
  - cairo=1.16.0=hd694305_1014
  - certifi=2022.9.14=py39haa95532_0
  - contourpy=1.0.5=py39h1f6ef14_0
  - cycler=0.11.0=pyhd8ed1ab_0
  - deepchem=2.6.1=pyhd8ed1ab_0
  - expat=2.4.9=h1537add_0
  - font-ttf-dejavu-sans-mono=2.37=hab24e00_0
  - font-ttf-inconsolata=3.000=h77eed37_0
  - font-ttf-source-code-pro=2.038=h77eed37_0
  - font-ttf-ubuntu=0.83=hab24e00_0
  - fontconfig=2.14.0=h720f74d_1
  - fonts-conda-ecosystem=1=0
  - fonts-conda-forge=1=0
  - fonttools=4.37.4=py39ha55989b_0
  - freetype=2.12.1=h546665d_0
  - gettext=0.19.8.1=h5728263_1009
  - greenlet=1.1.3=py39h415ef7b_0
  - icu=70.1=h0e60522_0
  - intel-openmp=2022.1.0=h57928b3_3787
  - joblib=1.2.0=pyhd8ed1ab_0
  - jpeg=9e=h8ffe710_2
  - kiwisolver=1.4.4=py39h2e07f2f_0
  - lcms2=2.12=h2a16943_0
  - lerc=4.0.0=h63175ca_0
  - libblas=3.9.0=16_win64_mkl
  - libbrotlicommon=1.0.9=h8ffe710_7
  - libbrotlidec=1.0.9=h8ffe710_7
  - libbrotlienc=1.0.9=h8ffe710_7
  - libcblas=3.9.0=16_win64_mkl
  - libdeflate=1.14=hcfcfb64_0
  - libffi=3.4.2=h8ffe710_5
  - libglib=2.74.0=h79619a9_0
  - libiconv=1.17=h8ffe710_0
  - liblapack=3.9.0=16_win64_mkl
  - libpng=1.6.38=h19919ed_0
  - libsqlite=3.39.4=hcfcfb64_0
  - libtiff=4.4.0=h8e97e67_4
  - libwebp-base=1.2.4=h8ffe710_0
  - libxcb=1.13=hcd874cb_1004
  - libzlib=1.2.12=hcfcfb64_3
  - m2w64-gcc-libgfortran=5.3.0=6
  - m2w64-gcc-libs=5.3.0=7
  - m2w64-gcc-libs-core=5.3.0=7
  - m2w64-gmp=6.1.0=2
  - m2w64-libwinpthread-git=5.0.0.4634.697f757=2
  - matplotlib-base=3.6.0=py39haf65ace_0
  - mkl=2022.1.0=h6a75c08_874
  - msys2-conda-epoch=20160418=1
  - munkres=1.1.4=pyh9f0ad1d_0
  - networkx=2.8.4=py39haa95532_0
  - numpy=1.23.3=py39h9061af7_0
  - openjpeg=2.5.0=hc9384bd_1
  - openssl=3.0.5=hcfcfb64_2
  - packaging=21.3=pyhd8ed1ab_0
  - pandas=1.5.0=py39h2ba5b7c_0
  - pcre2=10.37=hdfff0fc_1
  - pillow=9.2.0=py39hcef8f5f_2
  - pip=22.2.2=pyhd8ed1ab_0
  - pixman=0.40.0=h8ffe710_0
  - pthread-stubs=0.4=hcd874cb_1001
  - pycairo=1.21.0=py39h1f09dad_1
  - pyparsing=3.0.9=pyhd8ed1ab_0
  - python=3.9.13=hcf16a7b_0_cpython
  - python-dateutil=2.8.2=pyhd8ed1ab_0
  - python_abi=3.9=2_cp39
  - pytz=2022.4=pyhd8ed1ab_0
  - rdkit=2022.03.5=py39hfadf033_0
  - reportlab=3.5.68=py39h12d40c5_1
  - scikit-learn=1.1.2=py39hfd4428b_0
  - scipy=1.9.1=py39h316f440_0
  - setuptools=65.4.1=pyhd8ed1ab_0
  - six=1.16.0=pyh6c4a22f_0
  - sqlalchemy=1.4.41=py39ha55989b_0
  - sqlite=3.39.4=hcfcfb64_0
  - tbb=2021.6.0=h91493d7_0
  - threadpoolctl=3.1.0=pyh8a188c0_0
  - tk=8.6.12=h8ffe710_0
  - tzdata=2022d=h191b570_0
  - ucrt=10.0.20348.0=h57928b3_0
  - unicodedata2=14.0.0=py39hb82d6ee_1
  - vc=14.2=hb210afc_7
  - vs2015_runtime=14.29.30139=h890b9b1_7
  - wheel=0.37.1=pyhd8ed1ab_0
  - xorg-libxau=1.0.9=hcd874cb_0
  - xorg-libxdmcp=1.1.3=hcd874cb_0
  - xz=5.2.6=h8d14728_0
  - zlib=1.2.12=hcfcfb64_3
  - zstd=1.5.2=h7755175_4
  - pip:
    - absl-py==1.2.0
    - aiohttp==3.8.3
    - aiosignal==1.2.0
    - async-timeout==4.0.2
    - attrs==22.1.0
    - cachetools==5.2.0
    - charset-normalizer==2.1.1
    - click==8.1.3
    - colorama==0.4.5
    - docker-pycreds==0.4.0
    - filelock==3.8.0
    - frozenlist==1.3.1
    - fsspec==2022.8.2
    - gitdb==4.0.9
    - gitpython==3.1.27
    - google-auth==2.12.0
    - google-auth-oauthlib==0.4.6
    - grpcio==1.49.1
    - huggingface-hub==0.10.0
    - idna==3.4
    - importlib-metadata==5.0.0
    - markdown==3.4.1
    - markupsafe==2.1.1
    - multidict==6.0.2
    - oauthlib==3.2.1
    - pathtools==0.1.2
    - promise==2.3
    - protobuf==3.19.6
    - psutil==5.9.2
    - pyasn1==0.4.8
    - pyasn1-modules==0.2.8
    - pydeprecate==0.3.2
    - pytorch-lightning==1.7.7
    - pyyaml==6.0
    - regex==2022.9.13
    - requests==2.28.1
    - requests-oauthlib==1.3.1
    - rsa==4.9
    - sentry-sdk==1.9.10
    - setproctitle==1.3.2
    - shortuuid==1.0.9
    - smmap==5.0.0
    - tensorboard==2.10.1
    - tensorboard-data-server==0.6.1
    - tensorboard-plugin-wit==1.8.1
    - tokenizers==0.12.1
    - torch==1.12.1+cu116
    - torchaudio==0.12.1+cu116
    - torchmetrics==0.9.3
    - torchvision==0.13.1+cu116
    - tqdm==4.64.1
    - transformers==4.22.2
    - typing-extensions==4.3.0
    - urllib3==1.26.12
    - wandb==0.13.3
    - werkzeug==2.2.2
    - yarl==1.8.1
    - zipp==3.8.1
prefix: C:\Users\nicka\anaconda3\envs\relso_env
